versions/scylla/3.26.7/patch.patch

```diff
@@ -0,0 +1,243 @@
+diff --git a/tests/integration/standard/test_client_warnings.py b/tests/integration/standard/test_client_warnings.py
+index 6d5e040e..79df77fe 100644
+--- a/tests/integration/standard/test_client_warnings.py
++++ b/tests/integration/standard/test_client_warnings.py
+@@ -28,7 +28,6 @@ def setup_module():
+ 
+ # Failing with scylla because there is no warning message when changing the value of 'batch_size_warn_threshold_in_kb'
+ # config")
+-@xfail_scylla('Empty warnings: TypeError: object of type \'NoneType\' has no len()')
+ class ClientWarningTests(unittest.TestCase):
+ 
+     @classmethod
+diff --git a/tests/integration/standard/test_cluster.py b/tests/integration/standard/test_cluster.py
+index 43356dbd..b27c1cd6 100644
+--- a/tests/integration/standard/test_cluster.py
++++ b/tests/integration/standard/test_cluster.py
+@@ -288,7 +288,6 @@ class ClusterTests(unittest.TestCase):
+ 
+         cluster.shutdown()
+ 
+-    @xfail_scylla("Failing with scylla because there is option to create a cluster with 'lower bound' protocol")
+     def test_invalid_protocol_negotation(self):
+         """
+         Test for protocol negotiation when explicit versions are set
+@@ -1226,7 +1225,6 @@ class ClusterTests(unittest.TestCase):
+ 
+     @greaterthanorequalcass30
+     @lessthanorequalcass40
+-    @incorrect_test()
+     def test_compact_option(self):
+         """
+         Test the driver can connect with the no_compact option and the results
+diff --git a/tests/integration/standard/test_custom_cluster.py b/tests/integration/standard/test_custom_cluster.py
+index 6cdfb8d1..46ec7fa1 100644
+--- a/tests/integration/standard/test_custom_cluster.py
++++ b/tests/integration/standard/test_custom_cluster.py
+@@ -26,7 +26,7 @@ def setup_module():
+     config_options = {'native_transport_port': 9046}
+     ccm_cluster.set_configuration_options(config_options)
+     # can't use wait_for_binary_proto cause ccm tries on port 9042
```

Comment:
Should you remove this comment now also?
What has changed that we can use it now?